code/modules/shuttle/on_move.dm

```diff
@@ -149,6 +165,9 @@ All ShuttleMove procs go here
 	parallax_movedir = new_parallax_dir
 	return TRUE
 
+/area/proc/lateShuttleMove()
```

Comment:
>on movable, turf and area

Why not on /atom?

Reply:
keeping with the other procs here there will be different args in each one, I *have* been working on combining them all into one lower level proc like that but it's not done yet